test/NinjaTools.FluentMockServer.Tests/FluentAPI/Builders/ValueObjects/FluentBodyBuilderTests.cs

```diff
@@ -0,0 +1,382 @@
+    }
+}";
+            // Act & Assert
+            Assert(expected, builder => builder.NotMatchingXmlSchema(content));
+        }
+
+
+        [Fact]
+        public void NotContainingSubstring()
+        {
+            // Arrange
+            const string content = "some_string";
+            const string expected = @"{
+    ""body"": {
+        ""not"": true,
+        ""type"": ""STRING"",
+        ""string"": ""some_string"",
+        ""subString"": true
+    }
+}";
+            // Act & Assert
+            Assert(expected, builder => builder.NotContainingSubstring(content));
+        }
+
+        [Fact]
+        public void WithoutExactContent()
+        {
+            // Arrange
+            const string content = "some_string";
+            const string expected = @"{
+    ""body"": {
+        ""not"": true,
+        ""type"": ""STRING"",
+        ""string"": ""some_string""
+    }
+}";
+            // Act & Assert
+            Assert(expected, builder => builder.WithoutExactContent(content));
+        }
+
+
+        [Fact]
+        public void Without_String_And_Content_Type()
+        {
+            // Arrange
+            const string content = "some_string";
+            const string contentType = "text/xml";
+            const string expected = @"{
+    'body': {
+        'not': true,
+        'type': 'STRING',
+        'string': 'some_string',
+        'contentType': 'text/xml'
+    }
+}";
+            // Act & Assert
+            Assert(expected, bodyBuilder => bodyBuilder.WithoutExactContent(content, contentType));
+        }
+
+        private void Assert(string expected, Action<IFluentBodyBuilder> factory)
+        {
+            var builder = new FluentBodyBuilder();
+            factory(builder);
+
+            var resultJToken = builder.Build();
+            var expectedJToken = JObject.Parse(expected);
+
+            var envelope = new JObject(new JProperty("body", "{CONTENT}"));
+
+            var body = envelope.Children()
+                .OfType<JProperty>()
+                .First(jp => jp.Name == "body").Value
+                .Value<JValue>();
+
+            body.Replace(resultJToken);
+
+            var result = envelope.ToString(Formatting.Indented);
+            _outputHelper.WriteLine($"Expected:\n\n{expected}\n");
+            _outputHelper.WriteLine($"Actual:\n\n{result}\n");
+
+            envelope.Should().BeEquivalentTo(expectedJToken);
+            //     JsonAssertionExtensions.Should(envelope).BeEquivalentTo(expectedJToken);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4722826)